src/sentry/snuba/discover.py

```diff
@@ -340,7 +340,9 @@ def query(
         )
         if not found:
             raise InvalidSearchQuery(
-                "Aggregates used in a condition must also be in the selected columns."
+                "Aggregate {} used in a condition but is not a selected column.".format(
```

Comment:
Should have a u prefix on this string so you don't get a UnicodeError on non asci user input.